maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -87,7 +88,7 @@ object MaestroSessionManager {
                 heartbeatFuture.cancel(true)
                 SessionStore.delete(sessionId, selectedDevice.platform)
 
-                if (!SessionStore.hasActiveSessions(sessionId, selectedDevice.platform)) {
+                if (SessionStore.activeSessions().isEmpty()) {
```

Comment:
This will always be `false` since _current_ session is present in this list

Reply:
Current session actually gets deleted, in line number 89. I validated this from logs.